README.rst

```diff
@@ -35,6 +35,8 @@ Command line tools using Hachoir parsers:
 * hachoir-strip: modify a file to remove metadata
 * hachoir-urwid: display the content of a binary file in text mode
 
+Installation instructions: https://github.com/vstinner/hachoir/blob/master/doc/install.rst
```

Comment:
I would prefer to link to the rendered documentation: http://hachoir.readthedocs.io/en/latest/install.html

Reply:
Done